README.md

```diff
@@ -58,6 +58,10 @@ echo "c::1" | nc -u -w0 localhost 1234
     * Enable the CMake `conan-debug` profile.
   * CLion > View > Tool Windows > Conan > (gear) > Conan Executable: `$PROJECT_HOME/venv/bin/conan`
 
+## Docker Builds
+
+The `spectatord` project also supports a platform-agnostic build. The only prerequisite is the installation of `Docker`. Please see the file `DOCKERBUILD.md` in the `docs` folder for more information
```

Comment:
We like to wrap lines at 100 characters, to make reading diffs on longer documents a little easier.

I like separating the Docker documents out, but let's make this a link to that doc, so you can click down into it.